src/main/java/anto/Task.java

```diff
@@ -0,0 +1,58 @@
+package anto;
+
+/**
+ * Class representing a Task.
+ */
+public class Task {
+    /**
+     * Description of task.
+     */
+    protected String description;
+
+    /**
+     * Boolean representing whether task is completed.
+     */
+    protected boolean isDone;
```

Comment:
Is there a reason why these variables are protected instead of private? For instance, you have setter methods which modify isDone, but they are all used in the same package, so are redundant since there is already direct access to these varaibles anyway.